client/dashboard/task-list/tasks/appearance.js

```diff
@@ -190,7 +190,7 @@ class Appearance extends Component {
 				key: 'import',
 				label: __( 'Import demo products', 'woocommerce-admin' ),
 				description: __(
-					'We’ll add some products that it will make it easier to see what your store looks like.',
+					'We’ll add some products that it will make it easier to see what your store looks like',
```

Comment:
@justinshreve (just checking) The first `it` in the sentence looks like it may have been left behind in an edit as both the following would flow better 

> We’ll add some products that will make it easier to see what your store looks like

> We’ll add some products so that it will make it easier to see what your store looks like

Reply:
Thanks for catching that! - went with `We’ll add some products that will make it easier to see what your store looks like`